src/Pagination/Pagination.tsx

```diff
@@ -33,27 +35,37 @@ export const Pagination: React.FunctionComponent<PaginationProps> = React.memo((
      * @param {number} offset The offset to show the numbers
      * @returns {Array<number>} An array of the pages that needs to be displayed
      */
-    function getList(value: number, size: number, offset: number): Array<number> {
+    function getList(value: number, length: number): Array<number> {
         const list: Array<number> = [];
-        const range: { min: number, max: number } = { min: 0, max: 0 };
-        if (value <= (Math.ceil(offset / 2))) {
-            for (let i = 1; i <= offset; i++) {
-                list.push(i);
-            }
-        } else if (value >= (size + 1 - (Math.ceil(offset / 2)))) {
-            range.min = size + 1 - offset;
-            range.max = size;
-            for (let i = range.min; i <= range.max; i++) {
-                list.push(i);
-            }
-        } else {
-            range.min = value + 1 - (Math.ceil(offset / 2));
-            range.max = range.min + offset - 1;
-            for (let i = range.min; i <= range.max; i++) {
-                list.push(i);
+
+        // generate the array
+        for (let i = 1; i <= pagingSize; i++) {
```

Comment:
For consistency consider deciding if you want to either send all outside values into this function as a param or just use the values directly.  Right now you're sending the `value` and `length` as params but `pagingSize` is being used directly. 

You can also consider building the list and setting it to an internal state within a React.useEffect which would run every time any of the values you need change. Then you would just render the list within the return. Either way you should clean it up a bit.